README.md

```diff
@@ -9,8 +9,13 @@ Idea from Nick Carter, initial implementation from Richard Smith.
     Converts one (or several) .ninja_log files into chrome's about:tracing format
 
     Usage:
-        rm $BUILDDIR/.ninja_log && ninja -C $BUILDDIR
+        ninja -C $BUILDDIR
         ninjatracing $BUILDDIR/.ninja_log > trace.json
 
-    (If you don't have time for a clean build, at least run
-    `ninja -C $BUILDDIR -t recompact` first.)
+    By default this will show build timing results for the most recent (possibly
+    incremental) build. To show build timing results for every target, whether
+    built in the last build or previously, use --showall. Note that this will
+    overlap multiple builds and will thus exaggerate build parallelism.
+
+    (When using --showall if you don't have time for a clean build, at least run
```

Comment:
"When using --showall, ideally `rm $BUILDDIR/.ninja_log` and do a clean build. If you don't have time..."